sklearn/datasets/_samples_generator.py

```diff
@@ -169,13 +173,32 @@ def make_classification(
         for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    return_X_y : bool, default=True
+        If True, the ``(data, target)`` instead of a Bunch object is returned.
```

Comment:
Since the names seem to be `X` and `y` in this case I think we should use these names. Using `data` and `target` generally aside from the place in the doc for the `doc (Bunch)` object, might cause a bit of confusion.
```suggestion
        If True, a tuple ``(X, y)`` instead of a Bunch object is returned.
```